owltorrent-network/src/main/java/edu/rice/owltorrent/network/messages/ChokeMessage.java

```diff
@@ -0,0 +1,30 @@
+package edu.rice.owltorrent.network.messages;
+
+import edu.rice.owltorrent.network.PeerMessage;
+import java.nio.ByteBuffer;
+import lombok.EqualsAndHashCode;
+import lombok.Value;
+
+/**
+ * Choke Message for Peer Message
+ *
+ * @author: Shijie Fan
+ */
+@Value
+@EqualsAndHashCode(callSuper = true)
+public class ChokeMessage extends PeerMessage {
```

Comment:
As written in #18, would strongly recommend writing a "payloadless message" that takes care of choke/unchoke/interested/uninterested all together.